float64[] q
float64[] qdot
float64[] fL
float64[] fR
float64[] u_bias
float64 Kp_gainFactor
float64 Kd_gainFactor
float64 fL_gainFactor
float64 fR_gainFactor
